lib/metanorma/cli/setup.rb

```diff
@@ -0,0 +1,76 @@
+require "fontist"
+require "fileutils"
+
+module Metanorma
+  module Cli
+    class Setup
+      def initialize(options)
+        @options = options
+        @font_name = options.fetch(:font)
+        @no_wait = options.fetch(:no_wait, false)
+
+        create_metanorma_fonts_directory
+      end
+
+      def self.run(options = {})
+        new(options).run
+      end
+
+      def run
+        font = Metanorma::Cli.fonts.grep(/#{font_name}/i)
+
+        if font.empty?
+          font_paths = download_font
+          copy_to_fonts(font_paths)
+        end
+      end
+
+      private
+
+      attr_reader :font_name, :options, :no_wait
+
+      def create_metanorma_fonts_directory
+        unless Metanorma::Cli.fonts_directory.exist?
+          FileUtils.mkdir_p(Metanorma::Cli.fonts_directory)
+        end
+      end
+
+      def metanorma_fonts_path
+        @metanorma_fonts_path ||= Metanorma::Cli.fonts_directory
+      end
+
+      def download_font
+        Fontist::Finder.find(font_name)
+      rescue Fontist::Errors::MissingFontError
+        ask_user_and_download_font(font_name)
+      end
+
+      def copy_to_fonts(fonts_path)
+        fonts_path.each do |font_path|
+          font_name = File.basename(font_path)
+          FileUtils.copy_file(font_path, metanorma_fonts_path.join(font_name))
+        end
+      end
+
+      def ask_user_and_download_font(font_name)
```

Comment:
I propose to add some `--no-interactive` or `--yes` flag to be able to use this command on CI (or other non-interactive environments)

Probably make sense to make this flag not specific to setup command, to be able to access it from any command (just in case, because we have UI.ask in other commands too)

Reply:
Agree!

Reply:
Yap, I was looking into it. I've added the an explicit option `--agree-to-terms` to specify users agreement, and if the user didn't provide anything but it requires download then it will ask again.

But for the test or other toolchain we can use this `--agree-to-terms` option.